### DISCUSSION TITLE
Plot result along a line in reporting

### DISCUSSION THREAD

**Discussion by SergioP1975** [nextfoam/baram#140] [Ideas]
Hi, could be added the option to plot a desired variable across a line (defined by two points) in the report options? This can be done on Paraview, but having directly in the report options could be great.

**Comment by kdg5424** [comment]
Hello @SergioP1975. Thank you for your suggestion.
I'll upload our development list and talk to developers.

Thank you very much

Best regards

Donggyu Kim

**Reply by SergioP1975** [reply]
Thank you!

**Reply by kdg5424** [reply]
You're welcome @SergioP1975!!!
If you have any other asks or inquiry, please don't hesitate to ask me.

Best regards

Donggyu Kim
